src/Controllers/FilterController.cs

```diff
@@ -84,16 +84,20 @@ public IActionResult SubjectWizardPost(ResultsFilter filter)
         private IActionResult FullTextPost(ResultsFilter filter)
         {
             filter = filter.WithoutLocation();
+            Boolean queryIsEmpty = string.IsNullOrWhiteSpace(filter.query);
+            List<Provider> suggestions = _api.GetProviderSuggestions(filter.query);
```

Comment:
Can we move to closer to usage as it will execute needless here

Reply:
I have sort of done this with the new revision.